src/App.js

```diff
@@ -8,6 +8,8 @@ import getToken from "./lib/token";
 import Form from "./Form/Form";
 import * as firebase from "./lib/firebase";
 
+// heck spiky rake
+
```

Comment:
Remove unnecessary comments!

Reply:
LOL that was mine and I'm so sorry I keep forgetting about it and it never dies